connectors/base/base.go

```diff
@@ -188,3 +190,8 @@ func (c *Connector) Shutdown() error {
 	}
 	return c.Next.Shutdown()
 }
+
+// Name returns the name of the connector
+func Name() string {
```

Comment:
Every connector needs this, not just those that implement base, so let's make this part of the Connector interface